package/MDAnalysis/analysis/msd.py

```diff
@@ -415,9 +444,54 @@ def _conclude_fft(self):  # with FFT, np.float64 bit prescision required.
                 or set fft=False"""
             )
 
+        dump_times = self.times
         positions = self._position_array.astype(np.float64)
         for n in tqdm(range(self.n_particles)):
             self.results.msds_by_particle[:, n] = tidynamics.msd(
                 positions[:, n, :]
             )
         self.results.timeseries = self.results.msds_by_particle.mean(axis=1)
+        self.results.delta_t_values = np.arange(self.n_frames) * [
+            dump_times[1] - dump_times[0]
+        ]
+
+    def _conclude_non_linear(self):
+
+        dump_times = self.times
+        n_frames = self.n_frames
+        n_atoms = self.n_particles
+        positions = self._position_array.astype(np.float64)
+
+        msd_dict = collections.defaultdict(
+            list
+        )  # Dictionary to collect MSDs: {Δt: [msd1, msd2, ...]}
```

Comment:
Put the comment above the line of code (instead of afterwards) so that the code gets formatted on a single line